### DISCUSSION TITLE
Anyone else unable to deploy producion because of cloud-init==20.4 not being found?

### DISCUSSION THREAD

**Discussion by thewhiterabbit** [saleor/saleor#6731] [Q&A]
Trying to deploy Saleor 2.11 production for an ultra low budget maker store.
Halted at this error...

```
localhost:~/saleor$ pip3 install -r requirements.txt
ERROR: Could not find a version that satisfies the requirement cloud-init==20.4
ERROR: No matching distribution found for cloud-init==20.4
```
I wish there was a real guide for deploying Saleor to production in current 2.11 version, but I think they want that $999/month so bad they keep low budget developers in the dark intentionally with vague answers like "read the docs", yet the docs only describe installing into dev environment.

I have yet to see anyone in any forum able to deploy Saleor 2.11 to production successfully.

**Comment by lack-of-gravitas** [comment]
I can't even find clear instructions to deploy to production...
